2015-10-12 04:18:58.618 [error] <0.688.0> gen_server ibot_events_srv_node_interaction terminated with reason: bad return value: ok
2015-10-12 04:18:58.618 [error] <0.688.0> CRASH REPORT Process ibot_events_srv_node_interaction with 0 neighbours exited with reason: bad return value: ok in gen_server:terminate/7 line 804
2015-10-12 04:18:58.619 [error] <0.686.0> Supervisor ibot_events_sup had child ibot_events_srv_node_interaction started with ibot_events_srv_node_interaction:start_link() at <0.688.0> exit with reason bad return value: ok in context child_terminated
2015-10-29 04:58:58.634 [error] emulator Error in process <0.702.0> on node 'core@alex-N550JK' with exit value: {[{reason,{case_clause,{aborted,{node_not_running,'core@alex-N550JK'}}}},{mfa,{ibot_webi_handler,websocket_handle,3}},{stacktrace,[{ibot_db_func,get_from_mnesia,2,[{file,"src/ibot_db_func.erl"},{line,51}]},{ibot_db_func_config... 


2015-10-29 04:58:58.635 [error] <0.543.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.702.0> exit with reason: {[{reason,{case_clause,{aborted,{node_not_running,'core@alex-N550JK'}}}},{mfa,{ibot_webi_handler,websocket_handle,3}},{stacktrace,[{ibot_db_func,get_from_mnesia,2,[{file,"src/ibot_db_func.erl"},{line,51}]},{ibot_db_func_config,get_node_info,1,[{file,"src/ibot_db_func_config.erl"},{line,117}]},{ibot_db_func_config,generate_nodes_info_to_list,2,[{file,"src/ibot_db_func_config.erl"},{line,138}]},{ibot_core_app,get_project_node_from_config,0,[{file,"src/ibot_core_app.erl"},{line,116}]},{ibot_webi_handler,websocket_handle,3,[{file,"src/ibot_webi_handler.erl"},{line,78}]},{cowboy_websocket,handler_call,7,[{file,"src/cowboy_websocket.erl"},{line,598}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,529}]}]},{msg,{text,<<"{\"getNodes\" : \"\"}">>}},{req,[{socket,#Port<0.931>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.702.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},58740}},{host,<<"localhost">>},{host_info,undefined},{port,10100},{path,<<"/websocket">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"host">>,<<"localhost:10100">>},{<<"connection">>,<<"Upgrade">>},{<<"pragma">>,<<"no-cache">>},{<<"cache-control">>,<<"no-cache">>},{<<"upgrade">>,<<"websocket">>},{<<"origin">>,<<"http://localhost:10100">>},{<<"sec-websocket-version">>,<<"13">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/39.0.2171.65 Chrome/39.0.2171.65 Safari/537.36">>},{<<"accept-encoding">>,<<"gzip, deflate, sdch">>},{<<"accept-language">>,<<"en-US,en;q=0.8">>},{<<"sec-websocket-key">>,<<"Z0iktxWRHvZTSKO1EDhgLQ==">>},{<<"sec-websocket-extensions">>,<<"permessage-deflate; client_max_window_bits">>}]},{p_headers,[{<<"sec-websocket-extensions">>,[{<<"permessage-deflate">>,[<<"client_max_window_bits">>]}]},{<<"upgrade">>,[<<"websocket">>]},{<<"connection">>,[<<"upgrade">>]}]},{cookies,undefined},{meta,[{websocket_compress,false},{websocket_version,13}]},{body_state,waiting},{multipart,undefined},{buffer,<<>>},{resp_compress,false},{resp_state,done},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined_state}],[{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,529}]}]}

2015-10-30 01:30:18.179 [error] emulator Error in process <0.831.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-10-30 01:32:39.421 [error] emulator Error in process <0.834.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-10-30 01:35:27.403 [error] emulator Error in process <0.842.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-10-30 01:40:51.353 [error] <0.652.0> gen_server ibot_nodes_srv_service terminated with reason: bad argument in call to erlang:list_to_atom(new_cpp_service) in ibot_db_func_services:register_server_service/3 line 54
2015-10-30 01:40:51.353 [error] <0.652.0> CRASH REPORT Process ibot_nodes_srv_service with 0 neighbours exited with reason: bad argument in call to erlang:list_to_atom(new_cpp_service) in ibot_db_func_services:register_server_service/3 line 54
2015-10-30 01:40:51.353 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_service started with ibot_nodes_srv_service:start_link() at <0.652.0> exit with reason bad argument in call to erlang:list_to_atom(new_cpp_service) in ibot_db_func_services:register_server_service/3 line 54 in context child_terminated
2015-11-01 04:41:49.761 [error] <0.651.0> gen_server ibot_nodes_srv_connector terminated with reason: {enoent,[{erlang,open_port,[{spawn_executable,false},[{line,1000},stderr_to_stdout,{args,["/home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp.py","first_cpp","alex-N550JK","core@alex-N550JK","ibot_nodes_srv_connector","ibot_nodes_srv_topic","ibot_nodes_srv_service","ibot_nodes_srv_ui_interaction","ibot_events_srv_node_interaction",jv]}]],[]},{ibot_nodes_srv_connector,run_node,1,[{file,"src/ibot_nodes_srv_connector.erl"},{line,245}]},{ibot_nodes_srv_connector,handle_cast,2,...},...]}
2015-11-01 04:41:49.762 [error] <0.651.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: {enoent,[{erlang,open_port,[{spawn_executable,false},[{line,1000},stderr_to_stdout,{args,["/home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp.py","first_cpp","alex-N550JK","core@alex-N550JK","ibot_nodes_srv_connector","ibot_nodes_srv_topic","ibot_nodes_srv_service","ibot_nodes_srv_ui_interaction","ibot_events_srv_node_interaction",jv]}]],[]},{ibot_nodes_srv_connector,run_node,1,[{file,"src/ibot_nodes_srv_connector.erl"},{line,245}]},{ibot_nodes_srv_connector,handle_cast,2,...},...]} in gen_server:terminate/7 line 804
2015-11-01 04:41:49.763 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.651.0> exit with reason {enoent,[{erlang,open_port,[{spawn_executable,false},[{line,1000},stderr_to_stdout,{args,["/home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp.py","first_cpp","alex-N550JK","core@alex-N550JK","ibot_nodes_srv_connector","ibot_nodes_srv_topic","ibot_nodes_srv_service","ibot_nodes_srv_ui_interaction","ibot_events_srv_node_interaction",jv]}]],[]},{ibot_nodes_srv_connector,run_node,1,[{file,"src/ibot_nodes_srv_connector.erl"},{line,245}]},{ibot_nodes_srv_connector,handle_cast,2,...},...]} in context child_terminated
2015-11-01 04:42:30.588 [error] emulator Error in process <0.768.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 04:43:28.306 [error] emulator Error in process <0.776.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 04:46:10.401 [error] <0.651.0> gen_server ibot_nodes_srv_connector terminated with reason: {enoent,[{erlang,open_port,[{spawn_executable,"/home/alex/iBotOS/iBotOS/tp/new_projectdevfirst_cpp./first_cpp"},[{line,1000},stderr_to_stdout,{args,["/home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp.py","first_cpp","alex-N550JK","core@alex-N550JK","ibot_nodes_srv_connector","ibot_nodes_srv_topic","ibot_nodes_srv_service","ibot_nodes_srv_ui_interaction","ibot_events_srv_node_interaction",jv]}]],[]},{ibot_nodes_srv_connector,run_node,1,[{file,"src/ibot_nodes_srv_connector..."},...]},...]}
2015-11-01 04:46:10.402 [error] <0.651.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: {enoent,[{erlang,open_port,[{spawn_executable,"/home/alex/iBotOS/iBotOS/tp/new_projectdevfirst_cpp./first_cpp"},[{line,1000},stderr_to_stdout,{args,["/home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp.py","first_cpp","alex-N550JK","core@alex-N550JK","ibot_nodes_srv_connector","ibot_nodes_srv_topic","ibot_nodes_srv_service","ibot_nodes_srv_ui_interaction","ibot_events_srv_node_interaction",jv]}]],[]},{ibot_nodes_srv_connector,run_node,1,[{file,"src/ibot_nodes_srv_connector..."},...]},...]} in gen_server:terminate/7 line 804
2015-11-01 04:46:10.402 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.651.0> exit with reason {enoent,[{erlang,open_port,[{spawn_executable,"/home/alex/iBotOS/iBotOS/tp/new_projectdevfirst_cpp./first_cpp"},[{line,1000},stderr_to_stdout,{args,["/home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp.py","first_cpp","alex-N550JK","core@alex-N550JK","ibot_nodes_srv_connector","ibot_nodes_srv_topic","ibot_nodes_srv_service","ibot_nodes_srv_ui_interaction","ibot_events_srv_node_interaction",jv]}]],[]},{ibot_nodes_srv_connector,run_node,1,[{file,"src/ibot_nodes_srv_connector..."},...]},...]} in context child_terminated
2015-11-01 04:55:21.301 [error] <0.637.0> Mnesia('core@alex-N550JK'): ** ERROR ** (core dumped to file: "/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/MnesiaCore.core@alex-N550JK_1446_332121_299955")
 ** FATAL ** mnesia_tm crashed: {"Cannot read schema","/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/Mnesia.core@alex-N550JK/schema.DAT",{error,{file_error,"/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/Mnesia.core@alex-N550JK/schema.DAT",eacces}}} state: [<0.638.0>]

2015-11-01 04:55:26.281 [error] <0.632.0> gen_server ibot_nodes_srv_ui_interaction terminated with reason: bad record project_info in ibot_nodes_srv_ui_interaction:handle_cast/2 line 53
2015-11-01 04:55:26.281 [error] <0.632.0> CRASH REPORT Process ibot_nodes_srv_ui_interaction with 0 neighbours exited with reason: bad record project_info in ibot_nodes_srv_ui_interaction:handle_cast/2 line 53 in gen_server:terminate/7 line 804
2015-11-01 04:55:26.281 [error] <0.628.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_ui_interaction started with ibot_nodes_srv_ui_interaction:start_link() at <0.632.0> exit with reason bad record project_info in ibot_nodes_srv_ui_interaction:handle_cast/2 line 53 in context child_terminated
2015-11-01 04:55:31.300 [error] <0.639.0> gen_server mnesia_monitor terminated with reason: killed
2015-11-01 04:55:31.300 [error] <0.639.0> CRASH REPORT Process mnesia_monitor with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2015-11-01 04:55:31.300 [error] <0.642.0> gen_server mnesia_recover terminated with reason: killed
2015-11-01 04:55:31.300 [error] <0.642.0> CRASH REPORT Process mnesia_recover with 0 neighbours exited with reason: killed in gen_server:terminate/7 line 804
2015-11-01 04:55:31.300 [error] <0.637.0> Mnesia('core@alex-N550JK'): ** ERROR ** mnesia_event got unexpected event: {'EXIT',<0.640.0>,killed}

2015-11-01 04:55:31.300 [error] <0.637.0> CRASH REPORT Process mnesia_event with 0 neighbours exited with reason: killed in gen_event:terminate_server/4 line 302
2015-11-01 04:55:31.301 [error] <0.634.0> CRASH REPORT Process <0.634.0> with 0 neighbours exited with reason: {killed,{mnesia_sup,start,[normal,[]]}} in application_master:init/4 line 133
2015-11-01 05:00:44.375 [error] emulator Error in process <0.773.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 05:00:51.371 [error] emulator Error in process <0.781.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 05:27:56.527 [error] <0.650.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:open_port({spawn_executable,"/bin/bash"}, [{line,1000},stderr_to_stdout,{args,["/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_cor...",...]}]) in ibot_core_func_cmd:execute_os_script_file/2 line 79
2015-11-01 05:27:56.527 [error] <0.650.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:open_port({spawn_executable,"/bin/bash"}, [{line,1000},stderr_to_stdout,{args,["/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_cor...",...]}]) in ibot_core_func_cmd:execute_os_script_file/2 line 79
2015-11-01 05:27:56.527 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.650.0> exit with reason bad argument in call to erlang:open_port({spawn_executable,"/bin/bash"}, [{line,1000},stderr_to_stdout,{args,["/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_cor...",...]}]) in ibot_core_func_cmd:execute_os_script_file/2 line 79 in context child_terminated
2015-11-01 05:28:48.626 [error] emulator Error in process <0.767.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 05:43:54.483 [error] <0.651.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:open_port({spawn_executable,"/bin/bash"}, [{line,1000},stderr_to_stdout,{args,"/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_core-..."}]) in ibot_core_func_cmd:execute_os_script_file/1 line 79
2015-11-01 05:43:54.483 [error] <0.651.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:open_port({spawn_executable,"/bin/bash"}, [{line,1000},stderr_to_stdout,{args,"/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_core-..."}]) in ibot_core_func_cmd:execute_os_script_file/1 line 79
2015-11-01 05:43:54.483 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.651.0> exit with reason bad argument in call to erlang:open_port({spawn_executable,"/bin/bash"}, [{line,1000},stderr_to_stdout,{args,"/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_core-..."}]) in ibot_core_func_cmd:execute_os_script_file/1 line 79 in context child_terminated
2015-11-01 05:45:21.190 [error] <0.765.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:open_port({spawn_executable,"/bin/bash"}, [{line,1000},stderr_to_stdout,{args,"/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_core-..."}]) in ibot_core_func_cmd:execute_os_script_file/1 line 79
2015-11-01 05:45:21.190 [error] <0.765.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:open_port({spawn_executable,"/bin/bash"}, [{line,1000},stderr_to_stdout,{args,"/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_core-..."}]) in ibot_core_func_cmd:execute_os_script_file/1 line 79
2015-11-01 05:45:21.191 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.765.0> exit with reason bad argument in call to erlang:open_port({spawn_executable,"/bin/bash"}, [{line,1000},stderr_to_stdout,{args,"/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_core-..."}]) in ibot_core_func_cmd:execute_os_script_file/1 line 79 in context child_terminated
2015-11-01 05:50:56.147 [error] <0.651.0> gen_server ibot_nodes_srv_connector terminated with reason: {system_limit,[{erlang,list_to_atom,["bash /home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_core-1/priv/cpp_node_run.sh /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp first_cpp alex-N550JK core@alex-N550JK ibot_nodes_srv_connector ibot_nodes_srv_topic ibot_nodes_srv_service ibot_nodes_srv_ui_interaction ibot_events_srv_node_interaction jv"],[]},{ibot_core_func_cmd,exec,1,[{file,"src/ibot_core_func_cmd.erl"},{line,17}]},{ibot_nodes_srv_connector,handle_cast,2,[{file,...},...]},...]}
2015-11-01 05:50:56.147 [error] <0.651.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: {system_limit,[{erlang,list_to_atom,["bash /home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_core-1/priv/cpp_node_run.sh /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp first_cpp alex-N550JK core@alex-N550JK ibot_nodes_srv_connector ibot_nodes_srv_topic ibot_nodes_srv_service ibot_nodes_srv_ui_interaction ibot_events_srv_node_interaction jv"],[]},{ibot_core_func_cmd,exec,1,[{file,"src/ibot_core_func_cmd.erl"},{line,17}]},{ibot_nodes_srv_connector,handle_cast,2,[{file,...},...]},...]} in gen_server:terminate/7 line 804
2015-11-01 05:50:56.148 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.651.0> exit with reason {system_limit,[{erlang,list_to_atom,["bash /home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_core-1/priv/cpp_node_run.sh /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp first_cpp alex-N550JK core@alex-N550JK ibot_nodes_srv_connector ibot_nodes_srv_topic ibot_nodes_srv_service ibot_nodes_srv_ui_interaction ibot_events_srv_node_interaction jv"],[]},{ibot_core_func_cmd,exec,1,[{file,"src/ibot_core_func_cmd.erl"},{line,17}]},{ibot_nodes_srv_connector,handle_cast,2,[{file,...},...]},...]} in context child_terminated
2015-11-01 05:57:21.356 [error] <0.651.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:open_port({spawn_executable,"/bin/bash"}, [{line,1000},stderr_to_stdout,{args,[["/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_c...",...]]}]) in ibot_core_func_cmd:execute_os_script_file/1 line 79
2015-11-01 05:57:21.356 [error] <0.651.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:open_port({spawn_executable,"/bin/bash"}, [{line,1000},stderr_to_stdout,{args,[["/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_c...",...]]}]) in ibot_core_func_cmd:execute_os_script_file/1 line 79
2015-11-01 05:57:21.356 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.651.0> exit with reason bad argument in call to erlang:open_port({spawn_executable,"/bin/bash"}, [{line,1000},stderr_to_stdout,{args,[["/home/alex/iBotOS/iBotOS/ibot_os/rel/ibot_os/lib/ibot_c...",...]]}]) in ibot_core_func_cmd:execute_os_script_file/1 line 79 in context child_terminated
2015-11-01 06:01:05.134 [error] emulator Error in process <0.780.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 06:03:19.255 [error] <0.650.0> gen_server ibot_nodes_srv_connector terminated with reason: no function clause matching os:validate1(jv) line 366
2015-11-01 06:03:19.255 [error] <0.650.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: no function clause matching os:validate1(jv) line 366 in gen_server:terminate/7 line 804
2015-11-01 06:03:19.255 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.650.0> exit with reason no function clause matching os:validate1(jv) line 366 in context child_terminated
2015-11-01 06:04:55.715 [error] <0.651.0> gen_server ibot_nodes_srv_connector terminated with reason: no function clause matching os:validate1(jv) line 366
2015-11-01 06:04:55.715 [error] <0.651.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: no function clause matching os:validate1(jv) line 366 in gen_server:terminate/7 line 804
2015-11-01 06:04:55.716 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.651.0> exit with reason no function clause matching os:validate1(jv) line 366 in context child_terminated
2015-11-01 06:05:58.344 [error] <0.651.0> gen_server ibot_nodes_srv_connector terminated with reason: no function clause matching os:validate1(jv) line 366
2015-11-01 06:05:58.344 [error] <0.651.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: no function clause matching os:validate1(jv) line 366 in gen_server:terminate/7 line 804
2015-11-01 06:05:58.344 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.651.0> exit with reason no function clause matching os:validate1(jv) line 366 in context child_terminated
2015-11-01 06:09:20.767 [error] emulator Error in process <0.769.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 06:12:36.114 [error] <0.651.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:12:36.114 [error] <0.651.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:12:36.114 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.651.0> exit with reason bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 06:13:27.031 [error] <0.775.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:13:27.031 [error] <0.775.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:13:27.032 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.775.0> exit with reason bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 06:15:06.016 [error] <0.790.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:15:06.016 [error] <0.790.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:15:06.017 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.790.0> exit with reason bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 06:15:11.036 [error] <0.805.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:15:11.037 [error] <0.805.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:15:11.037 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.805.0> exit with reason bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 06:16:05.328 [error] emulator Error in process <0.771.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 06:16:17.058 [error] <0.650.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:16:17.058 [error] <0.650.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:16:17.058 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.650.0> exit with reason bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 06:16:31.356 [error] <0.776.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:16:31.356 [error] <0.776.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:16:31.356 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.776.0> exit with reason bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 06:16:32.949 [error] <0.778.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:16:32.949 [error] <0.778.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:16:32.949 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.778.0> exit with reason bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 06:16:33.920 [error] <0.780.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:16:33.920 [error] <0.780.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:16:33.921 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.780.0> exit with reason bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 06:16:34.710 [error] <0.782.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:16:34.711 [error] <0.782.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:16:34.711 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.782.0> exit with reason bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 06:16:35.527 [error] <0.784.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:16:35.527 [error] <0.784.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:16:35.527 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.784.0> exit with reason bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 06:16:37.828 [error] emulator Error in process <0.793.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 06:16:39.686 [error] <0.786.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:16:39.686 [error] <0.786.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:16:39.686 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.786.0> exit with reason bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 06:16:39.686 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.786.0> exit with reason reached_max_restart_intensity in context shutdown
2015-11-01 06:26:29.371 [error] <0.650.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:26:29.371 [error] <0.650.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:26:29.371 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.650.0> exit with reason bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 06:28:00.374 [error] <0.774.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:28:00.375 [error] <0.774.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:28:00.375 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.774.0> exit with reason bad argument in call to erlang:spawn("\e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp\n[su...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 06:29:50.722 [error] emulator Error in process <0.766.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 06:30:09.442 [error] emulator Error in process <0.773.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 06:30:16.819 [error] emulator Error in process <0.777.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 06:31:00.566 [error] emulator Error in process <0.767.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 06:34:18.288 [error] emulator Error in process <0.769.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 06:39:30.632 [error] emulator Error in process <0.769.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 06:46:42.781 [error] emulator Error in process <0.766.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 06:46:47.101 [error] emulator Error in process <0.769.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 06:50:50.760 [error] emulator Error in process <0.769.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 06:57:53.596 [error] <0.651.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("[sudo] password for alex: \e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nod...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:57:53.596 [error] <0.651.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("[sudo] password for alex: \e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nod...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 06:57:53.597 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.651.0> exit with reason bad argument in call to erlang:spawn("[sudo] password for alex: \e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nod...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 16:22:12.382 [error] <0.651.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("[sudo] password for alex: \e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nod...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 16:22:12.382 [error] <0.651.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("[sudo] password for alex: \e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nod...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 16:22:12.382 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.651.0> exit with reason bad argument in call to erlang:spawn("[sudo] password for alex: \e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nod...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 16:22:26.838 [error] <0.773.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("[sudo] password for alex: \e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nod...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 16:22:26.838 [error] <0.773.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("[sudo] password for alex: \e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nod...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 16:22:26.838 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.773.0> exit with reason bad argument in call to erlang:spawn("[sudo] password for alex: \e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nod...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 16:22:56.302 [error] <0.798.0> gen_server ibot_nodes_srv_connector terminated with reason: bad argument in call to erlang:spawn("[sudo] password for alex: \e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nod...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 16:22:56.302 [error] <0.798.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: bad argument in call to erlang:spawn("[sudo] password for alex: \e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nod...") in ibot_nodes_srv_connector:handle_cast/2 line 82
2015-11-01 16:22:56.302 [error] <0.649.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.798.0> exit with reason bad argument in call to erlang:spawn("[sudo] password for alex: \e[H\e[2Jstart cpp node /home/alex/iBotOS/iBotOS/tp/new_project/dev/nod...") in ibot_nodes_srv_connector:handle_cast/2 line 82 in context child_terminated
2015-11-01 16:29:10.211 [error] emulator Error in process <0.766.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 16:29:12.701 [error] emulator Error in process <0.769.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 16:41:18.853 [error] emulator Error in process <0.774.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 16:46:10.217 [error] emulator Error in process <0.765.0> on node 'core@alex-N550JK' with exit value: {{badrecord,service_server},[{ibot_nodes_srv_service,'-handle_info/2-fun-0-',6,[{file,"src/ibot_nodes_srv_service.erl"},{line,77}]}]}


2015-11-01 16:49:19.973 [error] <0.650.0> gen_server ibot_nodes_srv_connector terminated with reason: {enoent,[{erlang,open_port,[{spawn_executable,'echo 'alex!@#' | sudo -S /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp'},[{line,1000},stderr_to_stdout,{args,["first_cpp","alex-N550JK","core@alex-N550JK","ibot_nodes_srv_connector","ibot_nodes_srv_topic","ibot_nodes_srv_service","ibot_nodes_srv_ui_interaction","ibot_events_srv_node_interaction",jv]}]],[]},{ibot_nodes_srv_connector,run_node,1,[{file,"src/ibot_nodes_srv_connector.erl"},{line,254}]},{ibot_nodes_srv_connector,...},...]}
2015-11-01 16:49:19.974 [error] <0.650.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: {enoent,[{erlang,open_port,[{spawn_executable,'echo 'alex!@#' | sudo -S /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp'},[{line,1000},stderr_to_stdout,{args,["first_cpp","alex-N550JK","core@alex-N550JK","ibot_nodes_srv_connector","ibot_nodes_srv_topic","ibot_nodes_srv_service","ibot_nodes_srv_ui_interaction","ibot_events_srv_node_interaction",jv]}]],[]},{ibot_nodes_srv_connector,run_node,1,[{file,"src/ibot_nodes_srv_connector.erl"},{line,254}]},{ibot_nodes_srv_connector,...},...]} in gen_server:terminate/7 line 804
2015-11-01 16:49:19.974 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.650.0> exit with reason {enoent,[{erlang,open_port,[{spawn_executable,'echo 'alex!@#' | sudo -S /home/alex/iBotOS/iBotOS/tp/new_project/dev/nodes/first_cpp/first_cpp'},[{line,1000},stderr_to_stdout,{args,["first_cpp","alex-N550JK","core@alex-N550JK","ibot_nodes_srv_connector","ibot_nodes_srv_topic","ibot_nodes_srv_service","ibot_nodes_srv_ui_interaction","ibot_events_srv_node_interaction",jv]}]],[]},{ibot_nodes_srv_connector,run_node,1,[{file,"src/ibot_nodes_srv_connector.erl"},{line,254}]},{ibot_nodes_srv_connector,...},...]} in context child_terminated
